inputs/simple_inner.json

```diff
@@ -29,6 +29,15 @@
         "image_id": "BUILT_IMAGE_ID"
       },
       "name": "all_rpm_packages"
+    },
+    {
+      "args": {
+        "imagestream": "{{IMAGESTREAM}}",
+        "docker_image_repo": "{{DOCKER_IMAGE_REPO}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false
+      },
+      "name": "import_image"
```

Comment:
Does the simple build type get all the rebuild trigger bits? Maybe that should only be in the prod type.